drivers/rkt/api.go

```diff
@@ -0,0 +1,20 @@
+package rkt
+
+import (
+	"net"
+)
+
+type Pod struct {
```

Comment:
Can you include a package comment with a link to the original source for these? Will make updating them in the future easier.